website_sale_product_minimal_price/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     'name': 'Website Sale Product Minimal Price',
     'summary': 'Display minimal price for products that has variants',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.0.1',
```

Comment:
Remember that this is done by the bot.

Reply:
Ok, But should i delete it?

Reply:
Well, not this time, unless you plan to put other PR over the same module.

Reply:
ok